website/app/routes/show.js

```diff
@@ -113,6 +113,10 @@ export default class ShowRoute extends Route {
 
   @action
   error() {
-    this.router.replaceWith('error');
+    if (this.error.status === '404') {
```

Comment:
this may be part of the problem, but i don't think our error on line 32 is actually triggering a `404`